backend/fms_core/resources/experiment_run.py

```diff
@@ -0,0 +1,281 @@
+        except ValidationError as e:
+            errors = e.update_error_dict(errors).copy()
+
+        # Getting experiment_type
+        try:
+            obj.experiment_type = ExperimentType.objects.get(workflow=self.experiment_type_name)
+        except Exception as e:
+            errors["experiment_type"] = ValidationError([f"No experiment type with workflow {self.experiment_type_name} could be found."], code="invalid")
+
+        # Getting container
+        barcode = get_normalized_str(data, "Experiment Container Barcode")
+        kind = get_normalized_str(data, "Experiment Container Kind")
+
+        if barcode and kind:
+            try:
+                obj.container, container_created = Container.objects.get_or_create(
+                    barcode=barcode,
+                    kind=kind,
+                    defaults={'comment': f"Automatically generated via experiment run template import on "
+                            f"{datetime.utcnow().isoformat()}Z",
+                              'name': get_normalized_str(data, "Experiment Container Barcode")},
+                )
+            except Exception as e:
+                errors["container"] = ValidationError([f"Could not create experiment container. Barcode and kind are existing and do not match. "], code="invalid")
+
+        # Getting instrument
+        instrument_name = get_normalized_str(data, "Instrument Name")
+        if instrument_name:
+            try:
+                obj.instrument = Instrument.objects.get(name=instrument_name)
+            except Exception as e:
+                errors["instrument"] = ValidationError([f"No instrument named {instrument_name} could be found."], code="invalid")
+
+
+        experiment_run_processes_by_protocol_id = {}
+        # Create processes for ExperimentRun
+        for protocol in self.protocols_dict.keys():
+            obj.process = Process.objects.create(protocol=protocol,
+                                                    comment="Experiment (imported from template)")
+            for subprotocol in self.protocols_dict[protocol]:
+                sp = Process.objects.create(protocol=subprotocol,
+                                            parent_process=obj.process,
+                                            comment="Experiment (imported from template)")
+                experiment_run_processes_by_protocol_id[subprotocol.id] = sp
+
+
+        # Create property values for ExperimentRun
+        for i, (property, value) in enumerate(data.items()):
+            # Slicing columns not containing properties
+            if i < self.protocols_starting_idx or i >= self.protocols_ending_idx:
+                pass
+            else:
+                property_type = self.property_types_by_name[property]
+                if value:
+                    process = experiment_run_processes_by_protocol_id[property_type.object_id]
+
+                    if type(value).__name__ in ('datetime','time'):
```

Comment:
time does not seem to differentiate between a time of day which could use a Z and a time delta which would not.